src/rippled/accountState.js

```diff
@@ -110,13 +111,30 @@ const getAccountState = (account, rippledSocket) => {
         escrows: data[1],
         paychannels: data[2],
         xAddress: decomposedAddress || undefined,
+        deleted: false,
       })),
     )
     .catch((error) => {
       // X-address:
       //   error.toString(): CustomError: account not found
       //   error.code: 404
-      log.error(error.toString())
+      log.error(error)
+      if (error.message === 'account not found') {
```

Comment:
Check for `actNotFound` instead of the human readable message.

Reply:
this is the error printed in the console: `RippledError {name: 'RippledError', message: 'account not found', code: 404, stack: 'RippledError: account not found\n    at http://localhost:3001/static/js/main.chunk.js:52261:13'}
code: 404
message: "account not found"
name: "RippledError"
stack: "RippledError: account not found\n    at http://localhost:3001/static/js/main.chunk.js:52261:13"
[[Prototype]]: Error
constructor: ƒ RippledError(message, code)
[[Prototype]]: Object
constructor: ƒ Error()
message: ""
name: "Error"
toString: ƒ toString()
length: 0
name: "toString"
arguments: (...)
caller: (...)
[[Prototype]]: ƒ ()
[[Scopes]]: Scopes[0]
[[Prototype]]: Object`
I don't see where ` actNotFound` is